src/platforms/common/usb.c

```diff
@@ -29,7 +29,7 @@
 #include "serialno.h"
 
 usbd_device *usbdev = NULL;
-uint16_t usb_config;
+uint16_t usb_config = 0;
```

Comment:
As per the standard objects with static storage are initialized to zero, no need to explicitly provide an initializer

Reply:
Understood, reverted.